app/static/config/plots.js

```diff
@@ -69,7 +69,20 @@ module.exports = {
                 radius: 5
             }
         },
-        scaleExtent: [1, 10]
+        // the minimum/maxium scale factor allowed when zooming into the scatterplot
+        zoomExtent: [1, 12],
+        // the colourscale to use when plotting overlays to the neighbourplot
+        // should be one of the new matplot scales: inferno, magma, plasma or viridis
+        // see: https://bids.github.io/colormap/
+        colourScale: 'viridis',
+        // the minimum and maximum z scores to display on the colourmap
+        // outside the range will be clamped
+        colourScaleExtent: [-4, 4]
```

Comment:
Yeah that makes sense.. but I'll try and implement your +/- abs(max(5th
percentile, 95th percentile)) idea first?

On Tue, Feb 16, 2016 at 6:56 PM, Juan Nunez-Iglesias <
notifications@github.com> wrote:

> In app/static/config/plots.js
> <https://github.com/microscopium/microscopium-ui/pull/78#discussion_r52977470>
> :
>
> > @@ -69,7 +69,20 @@ module.exports = {
> >                  radius: 5
> >              }
> >          },
> > -        scaleExtent: [1, 10]
> > +        // the minimum/maxium scale factor allowed when zooming into the scatterplot
> > +        zoomExtent: [1, 12],
> > +        // the colourscale to use when plotting overlays to the neighbourplot
> > +        // should be one of the new matplot scales: inferno, magma, plasma or viridis
> > +        // see: https://bids.github.io/colormap/
> > +        colourScale: 'viridis',
> > +        // the minimum and maximum z scores to display on the colourmap
> > +        // outside the range will be clamped
> > +        colourScaleExtent: [-4, 4]
>
> I would say make this -3, 3. Anything more doesn't make sense.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/microscopium/microscopium-ui/pull/78/files#r52977470>.
>